README.md

```diff
@@ -11,6 +11,37 @@ It is backed by [hmpps-incident-reporting-api](https://github.com/ministryofjust
 
 The application needs a suite of services to work.
 
+### Using services in `dev` environment
+
+This is the easiest way to run and develop on your machine: by hooking into services that already exist
+in the `dev` environment.
+
+Run the application in development mode:
+
+* in separate shell sessions, for local development
+  * This will automatically restart it if server code or front-end assets are modified.
+
+### Using only local services
+
+```shell
+docker compose -f docker-compose.yml up -d
+npm run start:dev-local
+```
+
+* all in one, for preview:
+
+```shell
+docker compose -f docker-compose.yml --profile include-frontend up
+```
+
```

Comment:
Maybe mention that the UI will be accessible on localhost:3000?
it is obvious when looking at the containers but could save a new user some time

Reply:
@jnioche-jd the support URL has been added

Reply:
I'm not using docker to run the app TBH - dev won't normally do that - they'll run the app in dev mode separately - hence why the default it to skip it